label_studio/ml_models/migrations/0011_thirdpartymodelversion_model_provider_connection.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 3.2.25 on 2024-09-12 21:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+from core.redis import start_job_async_or_sync
+from ml_models.models import ThirdPartyModelVersion
+from ml_model_providers.models import ModelProviderConnection, ModelProviders
+
+
+def _fill_model_version_model_provider_connection():
+    third_party_model_versions = ThirdPartyModelVersion.objects.all()
+    for model_version in third_party_model_versions:
+        if model_version.provider == ModelProviders.OPENAI:
+            model_provider_connection = ModelProviderConnection.objects.filter(
+                organization=model_version.organization,
+                provider=model_version.provider
+            ).first()
+            model_version.model_provider_connection = model_provider_connection
+            model_version.save()
+        elif model_version.provider == ModelProviders.AZURE_OPENAI:
+            model_provider_connection = ModelProviderConnection.objects.filter(
+                organization=model_version.organization,
+                provider=model_version.provider,
+                deployment_name=model_version.provider_model_id,
+            ).first()
+            model_version.model_provider_connection = model_provider_connection
+            model_version.save()
+        else:
+            pass
+
+
+def forwards(apps, schema_editor):
+    start_job_async_or_sync(_fill_model_version_model_provider_connection)
+
+
+def backwards(apps, schema_editor):
```

Comment:
how about backward migrations? @farioas will be 😞😞😞

Reply:
Dont need them in this case since if we revert this migration, the column is just removed